src/App.jsx

```diff
@@ -1,12 +1,43 @@
 import { useState } from "react";
+import { Button } from "./Button.jsx";
+import { StatisticLine } from "./StatisticLine.jsx";
+import { Anecdote } from "./Anecdote.jsx";
 
 const App = () => {
-  // save each button click in its own state
   const [good, setGood] = useState(0);
   const [neutral, setNeutral] = useState(0);
   const [bad, setBad] = useState(0);
 
-  return <div>code here</div>;
+  const total = good + neutral + bad;
+  const average = total === 0 ? 0 : (good - bad) / total;
+  const positivePercentage = total === 0 ? 0 : (good / total) * 100;
+
+  console.log(total);
+  return (
+    <div>
+      <div>
+        <h2>Anecdotas</h2>
+        <Anecdote />
+      </div>
+
+      <h2>Give Feedback</h2>
+      <Button text="good" handleClick={() => setGood(good + 1)} />
+      <Button text="neutral" handleClick={() => setNeutral(neutral + 1)} />
+      <Button text="bad" handleClick={() => setBad(bad + 1)} />
+
+      <h2>Statistics</h2>
+      <table>
```

Comment:
> 1.9: unicafe step4
Change your application to display statistics only once feedback has been gathered.

You are rendering your table even if no feedback is still given.

FIX: spot the bug and fix it